src/components/home/modals/NodesManagement.vue

```diff
@@ -162,27 +170,38 @@ export default {
             Vue.set(userInfo, fieldName, value)
             window.localStorage.setItem(this.defaultAddress, JSON.stringify(userInfo))
         },
-        addNode() {
+        async addNode() {
             let nodeList = this.nodeList
             this.nodeURL = this.nodeURL.replace(/\s*/g, '')
             if (this.nodeURL in nodeList) {
                 return
             }
             this.init = true
             let suffix = '/addresses/balance/' + this.defaultAddress
-            this.$http.get(this.nodeURL + suffix).then(res => {
-                if (res.ok && res.body.address === this.defaultAddress) {
-                    Vue.set(nodeList, this.nodeURL, 'wallet')
+            let transactionsSuffix = '/transactions/list?address=' + this.defaultAddress + '&limit=10' + '&txType=' + PAYMENT_TX
+            let nodeType = 'Wallet'
```

Comment:
The node type should be changed. All nodes can be called full nodes because it has full block data. In the mean time, the node config with a customer API `/transactions/list` is a special full node, and we call it `Wallet Node` since only wallet will using `/transactions/list`.

<img width="499" alt="Screenshot 2021-01-19 at 9 32 57 PM" src="https://user-images.githubusercontent.com/41148850/105041780-61f96900-5a9e-11eb-9d0d-5c41ab011a84.png">